src/external/rawspeed/data/cameras.xml

```diff
@@ -5091,6 +5091,34 @@
 			<Alias>DMC-G70</Alias>
 		</Aliases>
 	</Camera>
+	<Camera make="Panasonic" model="DMC-G8">
+		<ID make="Panasonic" model="DMC-G8">Panasonic DMC-G8</ID>
+		<CFA width="2" height="2">
+			<Color x="0" y="0">BLUE</Color>
+			<Color x="1" y="0">GREEN</Color>
+			<Color x="0" y="1">GREEN</Color>
+			<Color x="1" y="1">RED</Color>
+		</CFA>
+		<Crop x="0" y="0" width="-208" height="0"/>
+		<Sensor black="143" white="4095"/>
+		<Aliases>
+			<Alias>DMC-G80</Alias>
```

Comment:
Hm, how do we know that `DMC-G8` is `DMC-G80`?
http://www.photographyblog.com/reviews/panasonic_lumix_dmc_g80_review/ says "DMC-G80 (also known as the Panasonic G85)"